starters/angular/basic/src/styles.scss

```diff
@@ -244,39 +244,12 @@ a:hover {
   color: var(--secondary-contrast);
   letter-spacing: -1%;
 }
-.link,
```

Comment:
@9kubczas4 we decided to keep the links CSS, can you revert? See https://chat.google.com/room/AAAA6WrZdos/QT6YN1rqe3E/7FKOjctpQuM?cls=10

Reply:
Oh, good catch! ;) Will do.

Reply:
Done. @leoortizz